app/src/main/java/com/bnyro/recorder/util/CustomLifecycleOwner.kt

```diff
@@ -0,0 +1,48 @@
+/*
+* Credits: https://gist.github.com/handstandsam/6ecff2f39da72c0b38c07aa80bbb5a2f
+*/
+
+package com.bnyro.recorder.util
+
+import android.os.Bundle
+import androidx.lifecycle.Lifecycle
+import androidx.lifecycle.LifecycleRegistry
+import androidx.savedstate.SavedStateRegistry
+import androidx.savedstate.SavedStateRegistryController
+import androidx.savedstate.SavedStateRegistryOwner
+
+internal class CustomLifecycleOwner : SavedStateRegistryOwner {
+    private var mLifecycleRegistry: LifecycleRegistry = LifecycleRegistry(this)
+    private var mSavedStateRegistryController: SavedStateRegistryController =
+        SavedStateRegistryController.create(this)
+
+    /**
+     * @return True if the Lifecycle has been initialized.
+     */
+    val isInitialized: Boolean
+        get() = true
```

Comment:
That's always true, so why do we have that? It's not used anywhere

Reply:
I wasn't sure about that. That code is from a GitHub gist by someone else (see the link on top of that file)